pkg/stanza/fileconsumer/file.go

```diff
@@ -182,82 +192,67 @@ func (m *Manager) consume(ctx context.Context, paths []string) {
 	m.roller.roll(ctx, readers)
 	m.saveCurrent(readers)
 	m.syncLastPollFiles(ctx)
+	m.clearCurrentFingerprints()
 }
 
-// makeReaders takes a list of paths, then creates readers from each of those paths,
+// makeReaders takes a lis of path, then creates reader,
```

Comment:
Typos in this comment, but more importantly, please review the entire comment as it no longer appears to describe the function.

Reply:
does it look ok now?